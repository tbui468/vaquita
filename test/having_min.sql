if exists drop database sol;
create database sol;
open sol;

create table planets (
                        id int8 key,
                        name text, 
                        mass float8, 
                        moons int8,
                        ring_system bool
                    );

insert into planets (
                        id,
                        name, 
                        mass, 
                        moons,
                        ring_system
                    ) 

values
        (1, "Venus", 2.32, 0, false),
        (2, "Earth", 5.0, 1, true),
        (3, "Mars", 4.23, 2, true),
        (4, "Saturn", 3.23, 2, true);

select ring_system, min(mass) from planets group by ring_system having min(mass) < 3.0;
select ring_system, min(mass) from planets group by ring_system having min(mass) > 3.0;
close sol;
drop database sol;
exit;
