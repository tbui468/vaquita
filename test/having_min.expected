created database sol
opened database sol
created table planets
inserted 4 record(s) into planets
ring_system, min(mass)
false, 2.320000
ring_system, min(mass)
true, 3.230000
closed database sol
dropped database sol
